client/dashboard/sites/backups/backup-details.tsx

```diff
@@ -38,34 +40,34 @@ export function BackupDetails( { backup, site }: { backup: ActivityLogEntry; sit
 		} );
 	};
 
+	const downloadButton = (
+		<Button variant="secondary" size="compact" icon={ download } onClick={ handleDownloadClick }>
+			{ __( 'Download backup' ) }
+		</Button>
+	);
+
+	const restoreButton = (
+		<Button variant="primary" size="compact" icon={ rotateLeft } onClick={ handleRestoreClick }>
+			{ __( 'Restore to this point' ) }
+		</Button>
+	);
+
+	const ActionContainer = isSmallViewport ? VStack : HStack;
+	const containerProps = isSmallViewport ? {} : { spacing: 2 };
+	const backupActions = backup.rewind_id ? (
+		<ActionContainer { ...containerProps }>
+			{ downloadButton }
+			{ restoreButton }
+		</ActionContainer>
+	) : null;
+
 	return (
 		<Card>
 			<CardHeader style={ { flexDirection: 'column', alignItems: 'stretch' } }>
 				<SectionHeader
 					title={ backup.summary }
 					decoration={ <Icon icon={ gridiconToWordPressIcon( backup.gridicon ) } /> }
-					actions={
-						backup.rewind_id && (
-							<HStack spacing={ 2 }>
-								<Button
-									variant="secondary"
-									size="compact"
-									icon={ download }
-									onClick={ handleDownloadClick }
-								>
-									{ __( 'Download backup' ) }
-								</Button>
-								<Button
-									variant="primary"
-									size="compact"
-									icon={ rotateLeft }
-									onClick={ handleRestoreClick }
-								>
-									{ __( 'Restore to this point' ) }
-								</Button>
-							</HStack>
-						)
-					}
+					actions={ backupActions }
```

Comment:
This works for this particular iteration, but looking ahead to the new mobile design proposal in BACKUP-264, we may need to either adjust the SectionHeader component to allow switching the actions between HStack and VStack, or move the actions out of the SectionHeader and place them at the same level inside the CardHeader so we can get additional flexibility.

Reply:
How about making it like below? I guess it's simpler (just add `wrap` to `HStack` inside the `SectionHeader` and looks better.

<img width="618" height="115" alt="image" src="https://github.com/user-attachments/assets/cfbdb1e0-35ae-4570-8f56-88f5682cf3bf" />

Reply:
This looks interesting, but I didn't get the same result. The `HStack` [is now a ButtonStack][1], so I added `wrap` to it. It rendered like:

<img width="421" height="226" alt="image" src="https://github.com/user-attachments/assets/d3d87a9f-0051-418b-9c3c-adb3bc25f792" />

[1]: https://github.com/Automattic/wp-calypso/blob/a2c759b75fb1f7bbb5baf45d927e7e285d297529/client/dashboard/sites/backups/backup-details.tsx#L50

Reply:
I got a result a bit better using `direction="column"`:

<img width="408" height="204" alt="image" src="https://github.com/user-attachments/assets/cf3ba369-7ce2-468a-bcee-3b8c97cc8c1a" />

I think we can use the viewport to decide between row/column and have a reasonable result.